gateway-messages/src/mgs_to_sp.rs

```diff
@@ -173,6 +173,28 @@ pub enum MgsRequest {
     /// Dump information about the lock state of the VPD (Vital Product Data)
     /// The values are serialized in the trailer of the packet
     VpdLockState,
+
+    /// Reset the SP after enabling the RoT watchdog
+    ///
+    /// This function is part of the SP reset sequence, and will return an error
+    /// if there is not a completed update in the SP.
+    ResetWithWatchdog {
```

Comment:
Do you think there's any chance we'll use a similar watchdog patterns for any other components? We originally had `ResetPrepare/ResetTrigger`, then later added `ResetComponentPrepare/ResetComponentTrigger` (with support for component `SP_ITSELF`), and I'm wondering if we should skip to that second step this time around.

Reply:
Yes, I switched over to component-flavored messages.